workflows/prognostic_c48_run/tests/test_monitor.py

```diff
@@ -0,0 +1,15 @@
+import xarray
+from runtime.monitor import Monitor
+from runtime.names import DELP
+
+
+def test_Monitor_monitor():
```

Comment:
```suggestion
def test_monitor_inserts_tendency_and_storage_of_one_var():
```

Reply:
Non-blocking since this is an improvement with any level of testing, but you should consider testing these cases:

- monitoring multiple variables
- monitoring with an empty set of variables

Given the inputs are so simple, you could also add assertions on the values.

Reply:
> Non-blocking since this is an improvement with any level of testing

I'd like to punt on this if it's okay.

Reply:
OK by me. I would still ask you update the name so it's clear this is a limited test.